src/platforms/common/enriching-events/screenshots.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: "Screenshots"
+sidebar_order: 106
+description: "Learn more about taking screenshots when an error occurs. Sentry pairs the screenshot with the original event, giving you additional insight into issues."
+supported:
+  - android
+---
+
+<Note>
+
+The Screenshots feature is available in Beta. Beta features are still in-progress and may have bugs. We recognize the irony.
+
+</Note>
+
+When a user experiences an error, Sentry provides the ability to take a screenshot and include as an [attachment](/platforms/android/enriching-events/attachments/).
```

Comment:
@imatwawana is there a way to link to `enriching-events/attachments` page but not from the `android` platform? rather redirecting to the current platform? this content will be available for other platforms as well.

Reply:
A PlatformLink component should work: 

<PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink>